src/ui/component/userEmailNew/view.jsx

```diff
@@ -39,12 +46,25 @@ function UserEmailNew(props: Props) {
           className="form-field--short"
           placeholder={__('hotstuff_96@hotmail.com')}
           type="email"
-          id="sign_up_email"
+          name="sign_up_email"
           label={__('Email')}
           value={newEmail}
           error={errorMessage}
           onChange={e => setEmail(e.target.value)}
         />
+        <FormField
+          type="checkbox"
+          name="sync_checkbox"
+          label={__('Sync your balance between devices')}
+          helper={
+            balance > 0
+              ? __("This is only available if you don't have a balance")
```

Comment:
This feature is not yet available for wallets with balances, but the gerbils are working on it.